docs/examples/modifying-the-head-element.md

```diff
@@ -0,0 +1,33 @@
+# Modifying Web Page Metadata via the ```<Head /> ```Element
```

Comment:
```suggestion
# Modifying Web Page Metadata via the `<Head />` Element
```
Triple backticks aren't needed for inline code snippets. Can you please use a single backtick where appropriate?